x-pack/plugins/security_solution/common/types/timeline/cells/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiDataGridCellValueElementProps } from '@elastic/eui';
+import { TimelineNonEcsData } from '../../../search_strategy';
+import { ColumnHeaderOptions } from '../columns';
+
+/** The following props are provided to the function called by `renderCellValue` */
+export type CellValueElementProps = EuiDataGridCellValueElementProps & {
```

Comment:
consider moving this to the package, because it overlaps with `x-pack/plugins/timelines/common/types/timeline/cells/index.ts`

Reply:
@kqualters-elastic will take care of it! :heart: